File: M5.Pt2/P8_Triangle_Nombre_Invers.py
#encoding: utf-8
#Programa que demana un nombre positiu més gran que 0 i mostra línia a línia del 1 fins al número introduït fent un triangle.
valor = int(input("Introdueix un valor: "))
print " "
for i in range (valor,0,-1):
	for o in range(1,i+1,1):
		print o,
	print " "

File: M5.Pt2/P8_Triangle_Nombre_Invers.java
import java.io.*;

/**
 * Programa que demana un nombre positiu més gran que 0 i mostra línia a línia del número introduït al 1 fent un triangle invers.
 **/
import java.util.*;
public class P8_Triangle_Nombre_Invers {
  public static void main(String[] args) throws IOException {
    BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
    String inputvalor;
    int valor,i,o;
    System.out.print("Introdueix un número: ");
    inputvalor = reader.readLine();
    valor = Integer.parseInt(inputvalor);
    System.out.println();
    for (i=valor;i>0;i--){
        for (o=1;o<=i;o++){
            if (o == i){
            System.out.println(o);
        }
        else{
            System.out.print(o+",");
        }
    }
  }
 }
}


